src/app/persoonsgegevens/templates/persoonsgegevens.mustache

```diff
@@ -54,17 +60,23 @@
                 <section>
                 <h2>Adresgegevens</h2>
                 <table class="responsive-table">
+                    <thead>
+                        <tr>
+                            <th>Kenmerk</th>
+                            <th>Waarde</th>
+                        </tr>
+                    </thead>
                     <tbody>
                         <tr>
-                            <th>Straat en huisnummer</th>
+                            <td>Straat en huisnummer</td>
                             <td>{{Persoon.Adres.Straat}} {{Persoon.Adres.Huisnummer}}</td>
```

Comment:
er missen nog adresvelden: huisnummer toevoeging en huisnummeraanduiding bestaan ook nog volgens mij (zie ook de IRMA-BRP api)

Reply:
In de API (testomgeving) krijg ik die velden niet langs, is er ergens een specificatie beschreven hiervan met mogelijke velden? Ik kan even niet vergelijken met Exxellence want daar is een storing :).